pages/doc/tags_overview.md

```diff
@@ -23,6 +23,7 @@ You use tags in several ways:
 * **Alert tags** -- Find [alerts](alerts.html) or exclude tagged alerts from a maintenance window.
 * **Event Tags** -- Add event tags from the Events browser or when you [create a user event](events.html#creating-a-user-event) to make it easier to filter events.
 * **Object tags** -- Limit the number of objects (e.g. dashboards) and metrics. For example, you might  display only dashboards with a certain tag.
+* **[Span tags](trace_data_details.html#span-tags)** -- Special tags that are associated with a span. Certain span tags are required for a span to be valid. An application can be instrumented to include custom span tags, as well.
```

Comment:
What do we mean by "special tags"? Is it good enough to say "tags"? 

An application can be instrumented to include custom span tags, as well. => In addition, you can instrument an application with custom span tags.

Reply:
+1